### ISSUE TITLE
JavaDoc jar shipped with the SDK is empty

### ISSUE THREAD

**Issue by Ndiritu** [microsoftgraph/msgraph-sdk-java#2078]
### Describe the bug

Empty JavaDoc JAR is hinders set-up time with the SDK.

What's missing from this upgrade guide is a link to _any_ v6 JavaDocs. The javadoc jar on [Maven Central](https://repo1.maven.org/maven2/com/microsoft/graph/microsoft-graph/6.13.0/) is empty, nothing but a `META-INF` directory. Ditto for _all_ the other v6 versions I checked. I'm trying to upgrade a v5 app and frustrated by the lack of JavaDocs. In particular, I'm stumbling around trying to find the `import` and API for classes like `PageIterator`, `RequestConfiguration `, etc. If you won't provide JavaDocs, then at least show the `import`s in your examples.

_Originally posted by @johnthad in https://github.com/microsoftgraph/msgraph-sdk-java/issues/2070#issuecomment-2209492155_
            

### Expected behavior

JavaDoc Jar to contain relevant docs

### How to reproduce

,

### SDK Version

6.x

### Debug output

<details><summary>Click to expand log</summary>
```

<Log output here>

```
</details>

**Comment by dias717** [comment]
@Ndiritu,

I agree with you. The JavaDoc does not work in Intellij. The reason is that the *-javadoc.jar file has only one manifest file in it. The other libraries in the *-javadoc.jar archive contain many files and weigh several megabytes. If I do the following **workaround**, then JavaDoc works in Intellij. Microsoft should do something similar and provide us with a working JavaDoc!


WORKAROUND TO USE THE JAVA DOC FOR MS GRAPH 6 in Intellij on Windows 10


**GENERATE JAVA-DOC:**

```
javadoc -d /c/Users/MY-USER-NAME/microsoft-graph-docs \
-sourcepath /c/Users/MY-USER-NAME/microsoft-graph-sources \
-subpackages com.microsoft.graph \
-classpath "/c/Users/MY-USER-NAME/.m2/repository/com/microsoft/kiota/microsoft-kiota-abstractions/1.7.0/microsoft-kiota-abstractions-1.7.0.jar:/c/Users/MY-USER-NAME/.m2/repository/com/microsoft/kiota/microsoft-kiota-serialization-json/1.7.0/microsoft-kiota-serialization-json-1.7.0.jar:/c/Users/MY-USER-NAME/.m2/repository/com/microsoft/kiota/microsoft-kiota-serialization-form/1.7.0/microsoft-kiota-serialization-form-1.7.0.jar:/c/Users/MY-USER-NAME/.m2/repository/com/microsoft/kiota/microsoft-kiota-serialization-multipart/1.7.0/microsoft-kiota-serialization-multipart-1.7.0.jar:/c/Users/MY-USER-NAME/.m2/repository/com/microsoft/kiota/microsoft-kiota-serialization-text/1.7.0/microsoft-kiota-serialization-text-1.7.0.jar:/c/Users/MY-USER-NAME/.m2/repository/jakarta/annotation/jakarta.annotation-api/3.0.0/jakarta.annotation-api-3.0.0.jar:/c/Users/MY-USER-NAME/.m2/repository/com/squareup/okhttp3/okhttp/4.12.0/okhttp-4.12.0.jar:/c/Users/MY-USER-NAME/.m2/repository/com/azure/azure-core/1.52.0/azure-core-1.52.0.jar:/c/Users/MY-USER-NAME/.m2/repository/com/microsoft/graph/microsoft-graph-core/3.3.1/microsoft-graph-core-3.3.1.jar"
```



**PACK THE GENERATED DATA INTO JAR:**

`cd /c/Users/MY-USER-NAME/microsoft-graph-docs`


`jar cf microsoft-graph-6.20.0-javadoc.jar *`



**COPY THE JAR INTO MS GRAPH FOLDER:**

`cp -f /c/Users/MY-USER-NAME/microsoft-graph-docs/microsoft-graph-6.20.0-javadoc.jar /c/Users/MY-USER-NAME/.m2/repository/com/microsoft/graph/microsoft-graph/6.20.0/`
